### PR TITLE
Update documentation for GitLab MR comments

### PR DESCRIPTION
**Pull request by chmccreery**
Screen Shot 2021-06-21 at 2 14 53 PM
Screen Shot 2021-06-21 at 2 15 06 PM

### CODE REVIEW

File: docs/notifications.md [hunk 1]
```diff
@@ -65,7 +69,46 @@ If you are using Github Actions to run Semgrep, no extra changes are needed to g
...
+
+<p style="text-align: center; font-size: 12px">
+    <img width="600px" src="../img/gitlab-mr-comment.png" alt="Screenshot of a GitLab MR comment"/><br/>
+    An inline GitLab merge request comment left by a custom Semgrep rule
+</p>
+
+To enable MR comments:
+
+1. Log into Semgrep's [Dashboard > Settings](https://semgrep.dev/manage/settings) to obtain your deployment ID and an API token.
+2. Examine your [policy settings](https://semgrep.dev/manage/policies/starter-policy?tab=settings) and make sure that PR comments are enabled. PR comments are the same thing as MR comments.
```
Comment (pabloest) [inline docs/notifications.md:84]:
```suggestion
2. Examine your [policy settings](https://semgrep.dev/manage/policies/starter-policy?tab=settings) and make sure that PR comments are enabled (MR comments and PR comments are enabled by the same toggle switch).
```

Note: this link didn’t load for me (I got a spinner but it never updated the main body of the page): https://semgrep.dev/manage/policies/starter-policy?tab=settings

File: docs/notifications.md [hunk 2]
```diff
@@ -65,7 +69,46 @@ If you are using Github Actions to run Semgrep, no extra changes are needed to g
...
+3. Create an API token on gitlab.com by going to [Profile > Access Tokens](https://gitlab.com/-/profile/personal_access_tokens) and adding a token with `api` scope.
+4. Copy the token that GitLab gives you.
+5. Navigate to your repository's CI/CD settings, scroll down to `Variables`, and click 'expand'. The url will look like https://gitlab.com/myusername/myproject/-/settings/ci_cd.
```
Comment (pabloest) [inline docs/notifications.md:87]:
```suggestion
5. Navigate to your repository's Settings > CI/CD, scroll down to 'Variables', and click 'Expand'. The url will look like https://gitlab.com/myusername/myproject/-/settings/ci_cd.
```

File: docs/notifications.md [hunk 3]
```diff
@@ -65,7 +69,46 @@ If you are using Github Actions to run Semgrep, no extra changes are needed to g
...
+6. Give the new variable the key `PAT` and use the token you copied in step 2 as the value. Select "mask variable" and **UNSELECT "protect variable"**.
```
Comment (pabloest) [inline docs/notifications.md:88]:
```suggestion
6. Click to 'Add variable', give the new variable the key `PAT` and use the token you copied in step 2 as the value. Select "mask variable" and **UNSELECT "protect variable"**.
```

**Review by pabloest — APPROVED** [review]

File: docs/notifications.md (hunk 1 above)
Comment (chmccreery) [reply docs/notifications.md:84]: @pabloest yes, this is a tough judgment call. New users get a policy called "starter-policy" but older users will have "getting-started" or something similar. We could instead here just link to semgrep.dev/manage/policies, which will work for everyone, but it won't bring them right to the policy settings page - they have to then click an individual policy and navigate to the "settings" tab.

Comment (pabloest) [reply docs/notifications.md:84]: @chmccreery ah makes sense. I’d leave the starter-policy then.

Comment (chmccreery) [reply docs/notifications.md:84]: In other words, I am happy to either stick with the link above, which will work for new users and not work for old users, or use /manage/policies, which will work for everyone but require two more clicks that may not be super obvious, so require more instructions here probably.

Comment (pabloest) [reply docs/notifications.md:84]: I’m in favor of optimizing for new users by keeping the link you’re using in this PR.

File: docs/notifications.md [hunk 4]
```diff
@@ -65,7 +69,46 @@ If you are using Github Actions to run Semgrep, no extra changes are needed to g
...
+
+<p style="text-align: center; font-size: 12px">
+    <img width="600px" src="../img/gitlab-mr-comment.png" alt="Screenshot of a GitLab MR comment"/><br/>
+    An inline GitLab merge request comment left by a custom Semgrep rule
+</p>
+
+To enable MR comments:
+
+1. Log into Semgrep's [Dashboard > Settings](https://semgrep.dev/manage/settings) to obtain your deployment ID and an API token.
+2. Examine your [policy settings](https://semgrep.dev/manage/policies/starter-policy?tab=settings) and make sure that PR comments are enabled (MR comments and PR comments are enabled by the same toggle switch).
```
Comment (chmccreery) [inline docs/notifications.md:84]:
```suggestion
2. Examine your policy settings by navigating to [Dashboard > Policies](https://semgrep.dev/manage/policies), selecting a policy, and switching to the Settings tab. Make sure that PR comments are enabled (MR comments and PR comments are enabled by the same toggle switch).
```

File: docs/notifications.md (hunk 1 above)
Comment (chmccreery) [reply docs/notifications.md]: Oops, somehow didn't see your comments before now! Feel free to change back!! Sorry @pabloest

Comment (chmccreery) [reply docs/notifications.md]: Oops, somehow didn't see your comments before now! Feel free to change back!! Sorry @pabloest

Comment (chmccreery) [reply docs/notifications.md]: https://github.com/returntocorp/semgrep-docs/pull/214

**Pull request merged**